contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -142,6 +142,7 @@ static SimpleEcontextStackEntry *simple_econtext_stack = NULL;
 	MemoryContextAlloc(get_eval_mcontext(estate), sz)
 #define eval_mcontext_alloc0(estate, sz) \
 	MemoryContextAllocZero(get_eval_mcontext(estate), sz)
+char *original_query_string;
```

Comment:
Let's initialize this to NULL. Otherwise, it could have a garbage value and `if (original_query)` may succeed.

Reply:
Initialized "original_query" to NULL.